packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md

```diff
@@ -23,8 +23,8 @@ If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source
 You will need the following things for this guide:
 
 - An Okta org - If you don't have an existing org, register for [Okta Developer Edition](https://developer.okta.com/signup/).
-- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard), under the "OpenID Connect Wizard" section.
-- At least one User [assigned to the Application](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps_Page.htm#Assigning).
+- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-attribute-statements-saml), under the "OpenID Connect Wizard" section.
+- At least one User [assigned to the Application](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-apps-page-show-application-embed-links).
```

Comment:
This is going to be changed shortly when I get time to refactor this page, but I'll take that as a todo to fix the link.

The text should be updated though to 
`- At least one user [assigned to the application]` (no need for caps as these aren't resource objects)